### PR TITLE
[HOTFIX] Fix subscribe to old subscriptions when re subscribing

### PR DESCRIPTION
**Pull request by jenow**
## What does this PR do ?

Fix an issue in the realtime controller where when we would subscribe then unsubscribe then subscribe again, the old subscribtions would subscribe again?

### CODE REVIEW

**Review by scottinet — CHANGES_REQUESTED** [review]
Missing tests?

**Comment by jenow** [comment]
> Missing tests?

There is no tests on this SDK as it comes from the community and we never had the time to add them

**Comment by scottinet** [comment]
Oh. I, uh... I see. I'll bring that issue in our internal meetings then.

**Review by scottinet — APPROVED** [review]

File: lib/src/controllers/realtime.dart [hunk 1]
```diff
@@ -12,34 +18,59 @@ import 'abstract.dart';
...
+    return 'Subscription: {index: ${index}, collection: ${collection}, filters: ${filters}, callback: ${callback}, autoResubscribe: ${autoResubscribe}, subscribeToSelf: ${subscribeToSelf}, state: ${state}, scope: ${scope}, users: ${users}, volatile: ${volatile}}';
+  }
+
+  @override
+  bool operator ==(other) {
+    return (other is Subscription) &&
+        other.index == index &&
+        other.collection == collection &&
+        // other.filters == filters &&
+        // other.callback == callback &&
```
Comment (Leodau) [inline lib/src/controllers/realtime.dart]: Forgotten comments?
```suggestion
```

**Review by Leodau — CHANGES_REQUESTED** [review]

**Review by Leodau — APPROVED** [review]

**Pull request merged**